Dockerfile

```diff
@@ -65,6 +65,7 @@ RUN echo "import sys\nsys.path.append('/source/python')" > /etc/python2.7/sitecu
 
 ADD tools/docker/supervisord.conf /etc/supervisor/conf.d/nav.conf
 
+COPY requirements/ /requirements
```

Comment:
COPY er nyere enn ADD. COPY kom fordi ADD har side-effects, f.eks ADD forstår og ka nlaste ned fra urler og kan pakke ut noen men ikke alle komprimerte filer noen ganger men ikke alltid gitt månefase og vanntemperatur etc. Jeg valgte COPY for å være eksplisitt med at det kun var kopiering jeg ville ha. https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy